hs/KernelPerceptron/BudgetPython.py

```diff
@@ -0,0 +1,165 @@
+#Python Implementation of the Budget Kernel Perceptron
```

Comment:
It seems like this implementation matches the Coq one pretty closely. I don't think that's necessarily a bad thing. But what are your thoughts on also comparing against a more highly optimized implementation of regular Perceptron, e.g., https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Perceptron.html? Would we learn anything more from that or is it not useful?

Reply:
The Scipy Perceptron implementation would be much more analogous to the regular Perceptron in MLCert as opposed to the Budget Kernel Perceptron. Because the Budget Kernel Perceptron's parameters are significantly larger than the parameters of regular Perceptron, I'm not sure how I would explain the time comparison. I looked in Scipy for an implementation of the Kernel Perceptron, since that'd be a much closer comparison, but I could not find an implementation in the documentation.